### ISSUE TITLE
Predict: Where do I publish?

### ISSUE THREAD

**Issue by bigfatnoob** [ai-se/citemap#33]
## Premise
Remember this table from the IST submission?
<img width="657" alt="screen shot 2017-03-01 at 4 52 07 pm" src="https://cloud.githubusercontent.com/assets/5582924/23483085/7109840c-fe9f-11e6-8896-56a462b717e6.png">
How can you figure out where you should be publishing?

## Experiment
In this experiment, I try to identify if we can predict which group(depending on time of the year when your paper is ready, you can target a conference from the group) should you probably submit a paper. 

### Attribute Selection
* LDADE: The topic distribution probability of each paper post LDADE is used as attributes.
* TFIDF: TFIDF is performed on the training data and the top 100 words are taken as attributes. (Why top 100? Cause if you use all the attributes, the classifier will become dead slow. And top 100 does not degrade the performance and also controls overfitting.)
* CFS: On the top 100 attributes selected by TFIDF, we select 20% of the top correlated attributes with the independent variables.

### Classification
* CART - Decision Trees 
* Logistic Regression

## Results

| Classifier      | Accuracy | F Score | Precision | Recall |
|-----------------|----------|-----------|--------|---------|
| LDADE + DecTree | 0.31 +- 0.01 | 0.32 +- 0.02 | 0.35 +- 0.02 | 0.35 +- 0.02 | 
| LDADE + LogReg  | 0.45 +- 0.02 | 0.42 +- 0.02 | 0.40 +- 0.02 | 0.45 +- 0.02 |
| LDADE + SVM     | 0.46 +- 0.01 | 0.42 +- 0.01 | 0.41 +- 0.03 | 0.46 +- 0.01 |
| LDADE + RF      | 0.38 +- 0.02 | 0.37 +- 0.01 | 0.38 +- 0.01 | 0.38 +- 0.02 |
| LDADE + NBayes  | 0.46 +- 0.00 | 0.29 +- 0.00 | 0.21 +- 0.00 | 0.46 +- 0.00 |
| TFIDF + DecTree | 0.48 +- 0.01 | 0.47 +- 0.01 | 0.47 +- 0.01 | 0.48 +- 0.01 |
| TFIDF + LogReg  | **0.55 +- 0.01** | 0.49 +- 0.01 | **0.54 +- 0.01** | **0.55 +- 0.01** |
| TFIDF + SVM     | **0.55 +- 0.01** | 0.48 +- 0.01 | 0.53 +- 0.02 | **0.55 +- 0.01** |
| TFIDF + RF      | **0.55 +- 0.01** | **0.52 +- 0.01** | 0.51 +- 0.03 | **0.55 +- 0.01** |
| TFIDF + NBayes  | 0.53 +- 0.01 | **0.52 +- 0.01** | 0.52 +- 0.02 | 0.53 +- 0.01 |
| CFS   + DecTree | 0.54+-0.02 | 0.50+-0.02 | 0.49+-0.02 | 0.54+-0.02 |
| CFS   + LogReg  | 0.54+-0.02 | 0.47+-0.02 | 0.49+-0.03 | 0.54+-0.02 |
| CFS   + SVM     | 0.54+-0.02 | 0.46+-0.02 | 0.48+-0.03 | 0.54+-0.02 |
| CFS   + RF      | 0.54+-0.02 | 0.50+-0.02 | 0.51+-0.02 | 0.54+-0.02 |
| CFS   + NB      | 0.55+-0.02 | 0.52+-0.03 | 0.50+-0.02 | 0.55+-0.02 |

## Conclusion
* TFIDF much better for attribute selection over LDADE.
* Regression is better than CART.
Reducing the set of vocabulary to 20 top words does not degrade the performance much.
